benchmark/misc/startup-cli-version.js

```diff
@@ -10,7 +10,7 @@ const path = require('path');
 // indispensible part of the CLI.
 const bench = common.createBenchmark(main, {
   cli: [
-    'tools/node_modules/eslint/bin/eslint.js',
+    'deps/npm/bin/npx-cli.js',
```

Comment:
I think we can simply test its existence in `fs.existsSync()` when creating the benchmark and only add it to the array when it exists (i.e. in dev mode).

Reply:
Yes, this could be another option here. I'll adjust this patch

Reply:
Done